src/components/NotFoundPage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const NotFound: React.FC = () => {
```

Comment:
```suggestion
export const NotFoundPage: React.FC = () => {
```

Reply:
![image](https://github.com/mate-academy/react_people-table-basics/assets/116891703/ff159347-8b4e-4fc2-b1e4-b71f06960ea9)


it's very strange because i chached it